src/Doctrine/Types/SerializedType.php

```diff
@@ -52,8 +52,15 @@ public function convertToPHPValue ($value, AbstractPlatform $platform)
 
         try
         {
+            $serializedValue = \base64_decode($value, true);
+
+            if (false === $serializedValue)
```

Comment:
```suggestion
            if (!\is_string($serializedValue))
```

Probably a bit more safe (and more in line with the required type.